ftplugin/go.vim

```diff
@@ -1,4 +1,10 @@
 if GOVIMPluginStatus() == "initcomplete"
+  " Highlights
+  highlight link govimErr Error
```

Comment:
Per our Slack conversation, this makes me a tiny bit nervous... because I don't think we can easily cater for everyone's respective themes. Hence whatever colours we choose here aren't ever going to be a great default for everyone. 

As we discussed however, the reason we need these values is:

* customisation in the case that folks want specific config for Go related errors/warnings
* the Vim highlight space doesn't have the option space, specifically it doesn't have definitions for warnings, info, hint etc

Reply:
Yeah, I agree. Let's monitor the feedback and see what users think about it. It is the best tradeoff I could come up with as a start.